crates/ty_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -241,15 +241,18 @@ class Bar(Foo[T_co], Generic[T_co]): ...
 
 static_assert(is_assignable_to(TypeOf[Bar[int]], type[Foo[int]]))
 static_assert(is_assignable_to(TypeOf[Bar[bool]], type[Foo[int]]))
-static_assert(is_assignable_to(TypeOf[Bar], type[Foo[int]]))
 static_assert(is_assignable_to(TypeOf[Bar[Any]], type[Foo[int]]))
+static_assert(is_assignable_to(TypeOf[Bar[Unknown]], type[Foo[int]]))
```

Comment:
Is this new `Unknown` case redundant with the `Any` case right above it, or are is their behavior different enough that it makes sense to test both?